### PR TITLE
Master product volume options dat

### PR DESCRIPTION
**Pull request by dat-odoo**
Task link : https://www.odoo.com/web?#id=1869002&action=327&model=project.task&view_type=form&menu_id=4720
Pad link : https://pad.odoo.com/p/r.f8a06c15faed51b168eb6ac8c5a01be1

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/product/models/res_config_settings.py [hunk 1]
```diff
@@ -28,6 +28,10 @@ class ResConfigSettings(models.TransientModel):
         ('0', 'Kilogram'),
         ('1', 'Pound'),
     ], 'Weight unit of measure', config_parameter='product.weight_in_lbs', default='0')
+    product_volume = fields.Selection([
+        ('0', 'Cubic Meters'),
+        ('1', 'Cubic Feet'),
```
Comment (rvalyi) [inline addons/product/models/res_config_settings.py]: lol

File: addons/product/models/product_template.py [hunk 2]
```diff
@@ -259,10 +257,21 @@ def _get_weight_uom_id_from_ir_config_parameter(self):
...
+
+    @api.depends('volume')
+    def _compute_volume_name(self):
+        """ Get the unit of measure to interpret the `volume` field. By default, we consider
+        that volumes are expressed in cubic meters. Users can configure to express them in cubic feet
+        by adding an ir.config_parameter record with "product.product_volume" as key
+        and "1" as value.
+        """
+        get_param = self.env['ir.config_parameter'].sudo().get_param
+        self.volume_name = "ft³" if get_param('product.volume') == '1' else "m³"
```
Comment (pimodoo) [inline addons/product/models/product_template.py]: could you do a helper like 'get_weight_uom_id_from_ir_config_parameter' for the volume and use the unit of measure.

**Comment by pimodoo** [comment]
@sle-odoo this one is ready to be merged

**Comment by sle-odoo** [comment]
robodoo r+

**Comment by wtaferner** [comment]
@sle-odoo @pimodoo 
Is there a rational why it is done like that? wouldn't it be better to bind it on the product itself by a uom relation instead of doing a global setting...it could even remain a global setting changing it by conversion.

It is master, so not yet completed I hope, but this is kind of weird from my PoV.

**Pull request closed without merging**